<ui version="4.0" >
 <class>RandRConfigBase</class>
 <widget class="QWidget" name="RandRConfigBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Display Configuration (X11 Resize, Rotate and Reflect)</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QWidget" native="1" name="outputList" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="identifyOutputsButton" >
         <property name="text" >
          <string>Identify Outputs</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGraphicsView" name="screenView" />
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
